docs/pages/admin-guides/api/getting-started.mdx

```diff
@@ -31,9 +31,8 @@ Here are the steps we'll walkthrough:
 Create a user `api-admin` with the built-in role `editor`:
 
 ```code
-# Run this directly on your Auth Server unless you are using Teleport Cloud
-# Add a user and login via the Proxy Service
-$ sudo tctl users add api-admin --roles=editor
+# Add a user and login with that user
```

Comment:
```suggestion
```
Since this comment repeats the line above, but also describes something (logging in) that isn't shown in the command.